pkg/apis/autoscaling/v2/defaults.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v2
+
+import (
+	autoscalingv2 "k8s.io/api/autoscaling/v2"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/kubernetes/pkg/apis/autoscaling"
+)
+
+var (
+	// These constants repeats previous HPA behavior
+	scaleUpLimitPercent         int32 = 100
+	scaleUpLimitMinimumPods     int32 = 4
+	scaleUpPeriod               int32 = 15
+	scaleUpStabilizationSeconds int32
+	maxPolicy                   = autoscalingv2.MaxChangePolicySelect
+	defaultHPAScaleUpRules      = autoscalingv2.HPAScalingRules{
+		StabilizationWindowSeconds: &scaleUpStabilizationSeconds,
+		SelectPolicy:               &maxPolicy,
+		Policies: []autoscalingv2.HPAScalingPolicy{
+			{
+				Type:          autoscalingv2.PodsScalingPolicy,
+				Value:         scaleUpLimitMinimumPods,
+				PeriodSeconds: scaleUpPeriod,
+			},
+			{
+				Type:          autoscalingv2.PercentScalingPolicy,
+				Value:         scaleUpLimitPercent,
+				PeriodSeconds: scaleUpPeriod,
+			},
+		},
+	}
+	scaleDownPeriod int32 = 15
+	// Currently we can set the downscaleStabilizationWindow from the command line
+	// So we can not rewrite the command line option from here
+	scaleDownStabilizationSeconds *int32 = nil
+	scaleDownLimitPercent         int32  = 100
+	defaultHPAScaleDownRules             = autoscalingv2.HPAScalingRules{
+		StabilizationWindowSeconds: scaleDownStabilizationSeconds,
```

Comment:
Nit: you can omit this and the variable, or just say `nil` if you want to be explicit.